### PR TITLE
data/journal_check/bug_refs.json: Typo got fixed for Tumbleweed

### PR DESCRIPTION
**Pull request by Vogtinator**
- Verification run: https://openqa.opensuse.org/tests/2635413

### CODE REVIEW

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -307,11 +307,17 @@
         "products": {
             "sle-micro": [ "5.0", "5.1", "5.2", "5.3" ],
             "leap-micro": ["5.2", "5.3"],
-            "microos":  ["Tumbleweed"],
```
Comment (DimStar77) [inline data/journal_check/bug_refs.json:310]: Yaiks - That makes all stagings fails from the time this was merged until the fixed health-checker is all around

Comment (Vogtinator) [reply data/journal_check/bug_refs.json:310]: True. I'll leave the other active temporarily.

**Review by DimStar77 — APPROVED** [review]

**Comment by jlausuch** [comment]
Why not having the generic one more flexible?
`"health-checker/fail.sh check\" failed|Machine didn't come up correct.*, do a rollback",`

**Comment by Vogtinator** [comment]
> Why not having the generic one more flexible? `"health-checker/fail.sh check\" failed|Machine didn't come up correct.*, do a rollback",`

To test (at least in the future) that the typo is gone.

**Review by jlausuch — APPROVED** [review]

**Comment by DimStar77** [comment]
Still turned out breaking existing products with the typo present

https://openqa.opensuse.org/tests/2638351#step/journal_check/21

**Comment by jlausuch** [comment]
> Still turned out breaking existing products with the typo present
> 
> https://openqa.opensuse.org/tests/2638351#step/journal_check/21

Also all SLE Micro maintenance jobs of today are also broken in the same place. I'd suggest to revert and find a solution.

File: data/journal_check/bug_refs.json [hunk 2]
```diff
@@ -312,6 +312,13 @@
         },
         "type": "ignore"
     },
+    "health-check": {
```
Comment (DimStar77) [inline data/journal_check/bug_refs.json:315]: The should be a unique name - the old every with the same babe is being overwritten

Comment (Vogtinator) [reply data/journal_check/bug_refs.json:315]: I should stop opening PRs that early I suppose.

Comment (DimStar77) [reply data/journal_check/bug_refs.json:315]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15489

will create VRs for old and new strings

**Pull request merged**